src/browser-lib/life-cycles.ts

```diff
@@ -0,0 +1,124 @@
+import { NgModuleRef, NgZone } from '@angular/core';
+
+import { SingleSpaAngularOpts, BootstrappedSingleSpaAngularOpts } from './types';
+import {
+  chooseDomElementGetter,
+  getContainerEl,
+  ivyEnabled,
+  removeApplicationFromDOMIfIvyEnabled,
+} from './dom';
+
+export async function bootstrap(opts: BootstrappedSingleSpaAngularOpts, props: any): Promise<void> {
```

Comment:
Moving all the code to a new file makes it a lot harder for me to review this effectively - I don't know what changed and what didn't.

I'd rather stick to the pattern that every other single-spa helper library uses, and keep these lifecycles in the main file.

Reply:
No problem. I will revert these changes to stick in the single file.